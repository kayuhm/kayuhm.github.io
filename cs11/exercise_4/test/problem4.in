3
17
15
1
16
12
-2
10
15
1