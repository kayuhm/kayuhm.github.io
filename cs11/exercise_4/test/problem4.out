Invalid Input!

16
14
Fizz

Buzz
11
Fizz
13
14
FizzBuzz

